packages/wonder-stuff-ci/src/get-mobile-release-logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from "winston";
+import * as gcpLogger from "@google-cloud/logging-winston";
+const {combine, timestamp, json, errors, printf, colorize} = winston.format;
+
+/**
+ * Winston logger used for generating to GCP for the mobile release process (release-raccoon).
+ * @param {string} projectId The GCP project ID to send logs to.
+ * @param {string} logName The name of the log to send logs to in GCP.
+ * @param {string} logLevel The minimum severity level to log.
+ * @param {boolean} enableConsoleLogs If true, logs will be sent to the console as well as GCP.
+ * @param {[key: string]: string} labels K/V metadata that will be add to the logs
```

Comment:
**suggestion:** Since there's only one parameter that is an object of options, you should create a type for that above the function (or in `types.ts`) and document each property in that type, then just use the type in a single `@param` here.

Reply:
ooo good point, i'll add that type!